### ISSUE TITLE
Will this efi compatible with cpu i5-8400

### ISSUE THREAD

**Issue by wtjin1992** [cewnhai/macOS-for-W650KJ-KK#1]
Hi there, my laptop is k680e from HASEE which is basically the same model as yours. I have updated and masked several pins to make it operate with a i5-8400 CPU. Do you have any idea that will my configuration be also able to work with your EFI folder? By the way, mine has a 1050ti dGPU which I suppose doesn't matter since it will be blocked anyway. Any help is appreciated

**Comment by cewnhai** [comment]
yep,Your k680e is w650kk,I update clover to use 10.14.5 ,U should try to use it 
   https://www.lanzous.com/i4zpnqd

**Comment by cewnhai** [comment]
> Hi there, my laptop is k680e from HASEE which is basically the same model as yours. I have updated and masked several pins to make it operate with a i5-8400 CPU. Do you have any idea that will my configuration be also able to work with your EFI folder? By the way, mine has a 1050ti dGPU which I suppose doesn't matter since it will be blocked anyway. Any help is appreciated

yep,Your k680e is w650kk,I update clover to use 10.14.5 ,U should try to use it 
   https://www.lanzous.com/i4zpnqd

**Comment by cewnhai** [comment]
> Hi there, my laptop is k680e from HASEE which is basically the same model as yours. I have updated and masked several pins to make it operate with a i5-8400 CPU. Do you have any idea that will my configuration be also able to work with your EFI folder? By the way, mine has a 1050ti dGPU which I suppose doesn't matter since it will be blocked anyway. Any help is appreciated

yep,Your k680e is w650kk,I update clover to use 10.14.5 ,U should try to use it 
   https://www.lanzous.com/i4zpnqd
